apps/web/src/components/layout/components/EnsureOnboardingComplete.tsx

```diff
@@ -6,9 +6,9 @@ import { useBlueprint } from '../../../hooks/index';
 export function EnsureOnboardingComplete({ children }: any) {
   useBlueprint();
   const location = useLocation();
-  const { claims } = useAuth();
+  const { currentOrganization, environmentId } = useAuth();
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** ‏ This doesn't build for me locally:
```
TS2339: Property 'environmentId' does not exist on type '{ inPublicRoute: boolean; inPrivateRoute: boolean; isLoading: boolean; currentUser: { organizationId: string; environmentId: string; _id: string; firstName?: string | undefined; lastName?: string | undefined; ... 7 more ...; hasPassword: boolean; }; ... 5 more ...; claims: IJwtClaims; }'.
     7 |   useBlueprint();
     8 |   const location = useLocation();
  >  9 |   const { currentOrganization, environmentId } = useAuth();
       |                                ^^^^^^^^^^^^^
    10 |
    11 |   if ((!currentOrganization || !environmentId) && location.pathname !== ROUTES.AUTH_APPLICATION) {
    12 |     return <Navigate to={ROUTES.AUTH_APPLICATION} replace />;


 ELIFECYCLE  Command failed with exit code 1.```

Reply:
Should be fixed now.